Google.Api.Generator/ProtoUtils/ResourceDetails.cs

```diff
@@ -156,13 +156,17 @@ public static IReadOnlyList<Definition> LoadResourceDefinitionsByFileName(IEnume
             // Compatibility problems: Adding a new resource with the same pattern as a single-pattern parent.
             //                         Adding a new resource with the same pattern as one of a multi-pattern parent.
             // TODO: Support new (Sept 2019) `name_descriptor` way of specifying resource-names.
-            var msgs = descs
+            var msgsFromProtoMsgs = descs
```

Comment:
Is the TODO here still valid?

Reply:
Yes. This is a separate annotation change that's currently still internal-only. I don't know when it's expected to become external.